includes/class-sensei-quiz.php

```diff
@@ -1487,6 +1486,11 @@ public static function start_quiz_questions_loop() {
 			true
 		);
 
+		// Display progress bar if pagination is enabled and progress bar is enabled.
+		if ( ! empty( $pagination_settings['pagination_number'] ) && array_key_exists( 'show_progress_bar', $pagination_settings ) && $pagination_settings['show_progress_bar'] ) {
+			self::display_progress_bar( $quiz_id, $pagination_settings );
+		}
+
```

Comment:
I'm not sure if this is the right place to display/load the progress bar. Maybe this logic should go in the progress bar method itself and trigger that via a hook? Also, to simplify things, you can fetch the pagination settings from inside that method instead of passing it down. Check out my other comment on the `display_progress_bar` method.

Reply:
Replied in the other questions

Reply:
KK did it :D